components/typography.css

```diff
@@ -0,0 +1,51 @@
+/* || Headings || */
+.h1,
```

Comment:
What is the use of giving styling on classname as well as the element?

Reply:
I think he will use that class in while using text utilities.

Reply:
Yeah! I was thinking of giving provision to use these styles in p tag, div tag etc